src/coverlet.core/coverlet.core.csproj

```diff
@@ -2,13 +2,14 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net472</TargetFrameworks>
     <AssemblyVersion>4.1.0</AssemblyVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Mono.Cecil" Version="0.10.1" />
     <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="System.Reflection.Metadata" Version="1.6.0" />
```

Comment:
```suggestion
    <PackageReference Condition="'$(TargetFramework)' == 'net472'" Include="System.Reflection.Metadata" Version="1.6.0" />
```

Reply:
💭 I don't think this change would have any real impact. It seems simpler to keep it like this?

Reply:
@ViktorHofer If we were creating a NuGet package from this and trying to avoid unnecessary declared dependencies in the result, I could see limiting this. However, that's not _currently_ the case. I can certainly make this change if it's needed or useful for some reason but wanted to verify first.

Reply:
Right we are not producing a package out of coverlet.core yet but IMO it's always a good thing to limit dependencies when introducing them. Others could keep that dependency even if we ship a package because they don't have context why this was added. As it's a one line change, pls do it 😁

Reply:
Fixed